fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/SizeStatisticsCollector.java

```diff
@@ -204,7 +178,7 @@ public long getMaxValueSize() {
      * @return the mean size (in bytes) of all keys
      */
     public double getAverageKeySize() {
-        return keySize * 1.0 / keyCount;
+        return sizeStatsResults.getKeySize() * 1.0 / sizeStatsResults.getKeyCount();
```

Comment:
These and others that do math should probably call `sizeStatsResult.equivalentMethod()`.

Reply:
oh yes, how lazy I was...